src/components/show-edit-pod/__spec__.js

```diff
@@ -226,50 +211,37 @@ describe('ShowEditPod', () => {
 
   describe('mainClasses', () => {
     it('returns the base class', () => {
-      expect(instance.mainClasses).toEqual('carbon-show-edit-pod')
+      expect(instance.mainClasses).toEqual('carbon-show-edit-pod');
     });
 
     it('returns any passed props', () => {
-      instance = TestUtils.renderIntoDocument(
-        <ShowEditPod
-          className='foo'
-        />
-      );
+      instance = TestUtils.renderIntoDocument(<ShowEditPod className='foo' />);
 
-      expect(instance.mainClasses).toEqual('carbon-show-edit-pod foo')
+      expect(instance.mainClasses).toEqual('carbon-show-edit-pod foo');
     });
   });
 
   describe('deleteButton', () => {
     it('renders a link', () => {
-      let deleteSpy = jasmine.createSpy('delete');
+      const deleteSpy = jasmine.createSpy('delete');
 
-      instance = TestUtils.renderIntoDocument(
-        <ShowEditPod
-          onDelete={ deleteSpy }
-        />
-      );
+      instance = TestUtils.renderIntoDocument(<ShowEditPod onDelete={ deleteSpy } />);
       instance.setState({ editing: true });
 
       TestUtils.findRenderedDOMComponentWithClass(instance, 'carbon-show-edit-pod__delete');
     });
 
     describe('when delete text is passed', () => {
       it('renders the custom text', () => {
-        let deleteSpy = jasmine.createSpy('delete');
+        const deleteSpy = jasmine.createSpy('delete');
 
-        instance = TestUtils.renderIntoDocument(
-          <ShowEditPod
-            onDelete={ deleteSpy }
-            deleteText='foo'
-          />
-        );
+        instance = TestUtils.renderIntoDocument(<ShowEditPod onDelete={ deleteSpy } deleteText='foo' />);
```

Comment:
prefer this longer line as it was previously

Reply:
done